### PR TITLE
Release 0.9.1

### PR DESCRIPTION
**Pull request by mikemhenry**
This PR will be used to update the change log and when merged will represent the 0.9.1 release.

### CODE REVIEW

**Comment by mikemhenry** [comment]
This release will also fix https://github.com/choderalab/perses/issues/784

**Comment by mikemhenry** [comment]
We can merge this after #830 gets merged

**Review by ijpulidos — APPROVED** [review]
Great work!

**Pull request merged**
